sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuParquetFileFormat.scala

```diff
@@ -131,7 +133,13 @@ object GpuParquetFileFormat {
     }
 
     SparkShimImpl.parquetRebaseWrite(sqlConf) match {
-      case "EXCEPTION" | "CORRECTED"| "LEGACY" => // Good
+      case "EXCEPTION" | "CORRECTED" => // Good
+      case "LEGACY" =>
+        val tz = TimeZone.getDefault.getID
```

Comment:
nit: can we make this code match the other timezone check code?

https://github.com/NVIDIA/spark-rapids/blob/feb0c86d4cdbed747335a1db4551422bbad926ca/sql-plugin/src/main/scala/com/nvidia/spark/rapids/TypeChecks.scala#L830-L842

I am not saying that you need to call these methods, but I am saying that we should use the same code to check the timestamp is UTC.